include/esp-stub-lib/bit_utils.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * SPDX-FileCopyrightText: 2025 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ */
+
+#pragma once
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifndef BIT
+#define BIT(nr)                 (1UL << (nr))
```

Comment:
about code style, should we comply some space standard here: alignment by the defined value, or just a space? Or stick to the same style only within one file?